modules/seqexec/server/src/main/scala/seqexec/server/gnirs/GnirsControllerEpics.scala

```diff
@@ -234,7 +235,8 @@ object GnirsControllerEpics extends GnirsController {
     val coaddsWriter = smartSetParam(config.coadds, epicsSys.numCoadds,
       dcCmd.setCoadds(config.coadds))
 
-    val biasWriter =smartSetDoubleParam(biasTolerance)(encode(config.wellDepth), epicsSys.detBias,
+    // Value read from the instrument is the negative of what was set
+    val biasWriter = smartSetDoubleParam(biasTolerance)(-encode(config.wellDepth), epicsSys.detBias,
```

Comment:
Are we sure this should change from `encode` to `-encode`?

Reply:
The arguments are: value too set, function to read back the applied value, function to apply the new value. The issue was that the value read back  from the instrument was the negative of the value set. Alternatively, I could have changed the sign of the second argument.